libcloud/storage/drivers/cloudfiles.py

```diff
@@ -310,15 +310,18 @@ def get_object(self, container_name, object_name):
 
         raise LibcloudError('Unexpected status code: %s' % (response.status))
 
-    def get_container_cdn_url(self, container):
+    def get_container_cdn_url(self, container, ssl_uri=False):
```

Comment:
our practice is that non-standard arguments are prefixed with `ex_` please can you rename this keyword argument to `ex_ssl_uri` to indicate that it's not part of the base class